tests/jsx-split-attrs/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -90,11 +90,21 @@ long_open_long_children = (
       Hello world
     </BaseForm>
     <div>
-      <div><div><div><div><div>hey hiya how are ya</div></div></div></div></div>
+      <div>
+        <div>
+          <div>
+            <div>
+              <div>hey hiya how are ya</div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
FWIW, the original behavior was very much intentional. 

The deep nesting is plenty visible in the inline case, and if it needs to be done, why make it even more annoying?

Especially with Styled Components and the like taking off, I don't think we'd want to effectively discourage things like this:

```
render() {
  return (
    <MyList>
      {this.props.items.map(item => (
        <MyListItem><Text bold green><MyDomainSpecific>{item}</MyDomainSpecific></Text></MyListItem>
      ))}
    </MyList>
  )
}